seacrowd/sea_datasets/indommlu/indommlu.py

```diff
@@ -0,0 +1,291 @@
+    "Bahasa Jawa": "Local languages and cultures",
+    "PPKN": "Social science",
+}
+
+special_case = ["SD-SMP-SMA", "SD-SMP"]
+level_mapper = {
+    "SMA": "SMA",  # SMA --> high school level"
+    "Seleksi PTN": "University entrance test",
+    "SD": "SD",  # SD --> elementary school level
+    "SMP": "SMP",  # SMP --> junior high school level
+    "Kelas I SD": "SD",
+    "Kelas X SMA": "SMA",
+    "Kelas XI SMA": "SMA",
+    "Kelas XII SMA": "SMA",
+    "V SD": "SD",
+    "VI SD": "SD",
+    "VII SMP": "SMP",
+    "VIII SMP ": "SMP",
+    "IX SMP": "SMP",
+    "Kelas III SD": "SD",
+    "Kelas IV SD": "SD",
+    "Kelas II SD": "SD",
+}
+
+
+def fix_level(level, kelas):
+    # Fixing Level
+    if level in special_case:
+        kelas = float(kelas)
+        if kelas >= 1 and kelas <= 6:
+            level = "SD"
+        elif kelas >= 7 and kelas <= 9:
+            level = "SMP"
+        elif kelas >= 10:
+            level = "SMA"
+        else:
+            print(level)
+    fixed_level = level_mapper[level]
+
+    # Fixing class
+    kelas = str(kelas)
+    if kelas.strip() in ["PTN", "2023-10-12 00:00:00"]:
+        fixed_kelas = 13
+    elif kelas == "4,5,6":
+        fixed_kelas = 6
+    else:
+        fixed_kelas = int(float(kelas.strip()))
+
+    # sanity check over the level and kelas
+    return fixed_level, fixed_kelas
+
+
+def pass_schema_filter(schema, row):
+    if schema == "source":
+        return True
+    lang = schema.split("_")[1]
+    if lang not in _LANGUAGES:  # seacrowd_qa
+        return True
+    if lang == "ind":  # contains "Bahasa Indonesia" and all other non-language subjects
+        return (lang2subject[lang] == row["subject"]) or (row["subject"] not in lang2subject.values())
+    return lang2subject[lang] == row["subject"]
+
+
+class IndoMMLUDataset(datasets.GeneratorBasedBuilder):
+    """IndoMMLU is the first multitask language understanding benchmark for Indonesian culture and languages."""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_source",
+            version=SOURCE_VERSION,
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=_DATASETNAME,
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_seacrowd_qa",
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd schema",
+            schema="seacrowd_qa",
+            subset_id=_DATASETNAME,
+        ),
+    ]
+    for lang in _LANGUAGES:
+        lang_config = SEACrowdConfig(
+            name=f"{_DATASETNAME}_{lang}_seacrowd_qa",
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} {lang} SEACrowd schema",
+            schema=f"seacrowd_{lang}_qa",
+            subset_id=_DATASETNAME,
+        )
+        BUILDER_CONFIGS.append(lang_config)
```

Comment:
please also add the source config for all languages
`name=f"{_DATASETNAME}_{lang}_source"`